src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+- To list descendent folders: [Org Administrator](authorization.html#org-administrator), [Folder Admin](authorization.html#folder-admin) or [Folder Mover](authorization.html#folder-mover).
+- To list descendent clusters: [Cluster Administrator](authorization.html#cluster-administrator), [Cluster Developer](authorization.html#cluster-developer), [Cluster Creator](authorization.html#cluster-mover), or [Cluster Operator](authorization.html#cluster-operator).
+
+{{site.data.alerts.callout_success}}
+To discover the entire folder structure, you must list each folder's contents separately.
+{{site.data.alerts.end}}
+
+1. To list the clusters and folders at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://cockroachlabs.cloud/api/v1/folders/root/contents' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace `<api-key>` with your service account's API key.
+
+1. To list the clusters and folders contained within another folder, specify its ID instead of `root`:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request GET \
+           --url 'https://cockroachlabs.cloud/api/v1/folders/<folder_id>/contents' \
+           --header 'Authorization: Bearer <api-key>'
+    ~~~
+
+    Replace:
+    - `<folder-id>`: The ID of a folder.
+    - `<api-key>`: Your service account's API key.
+
+{{site.data.alerts.callout_info}}
+In the API, each folder is referred to by its ID, regardless of where it is located in the hierarchy.
+{{site.data.alerts.end}}
+
+## Create a cluster in a folder
+
+Your service account must have the following roles on the organization or the folder:
+
+- [Cluster Administrator](authorization.html#cluster-administrator) or [Cluster Creator](authorization.html#cluster-creator)
+
+To create a cluster in a folder, set the `parent_id` field to the ID of the folder.
+
+{{site.data.alerts.callout_info}}
+If you do not specify a `parent_id`, or if you set `parent_id` to `root`, the cluster is created at the level of the organization.
+{{site.data.alerts.end}}
+
+For example, to create a {{ site.data.products.serverless }} deployed on GCP in a folder:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+curl --request POST \
+       --url 'https://cockroachlabs.cloud/api/v1/clusters' \
+       --header 'Authorization: Bearer <api-key>' \
+       --header 'content-type: application/json' \
+       --data '{"name":"<cluster_name>","provider":"GCP","spec":{"serverless":{"regions":["us-central1"],"spend_limit":0}, "parent_id":"<parent_id>"}}'
+~~~
+
+Replace:
+
+- `<cluster_name>`: A name for the cluster. Replace `<parent_id>` with the ID of the folder.
+- `<api-key>`: Your service account's API key.
+- `<parent-id>`: The ID of a folder.
+
+## Move a cluster into or out of a folder
+
+Your service account must have the following roles on the organization or on both the source and destination locations:
```

Comment:
nit: maybe replace "roles" with "the move permission", which is via the Folder Admin or Folder Mover role

Reply:
I think "the move permission" sounds more mysterious than just listing which roles are sufficient to do the action. WDYT? Is it incorrect as written?

Reply:
AFAIK IAM docs usually discuss not just roles but permissions as well. I see no harm in it, but this is a nitpick so I'll leave it to you

Reply:
Done